lib/src/commands/LayoutTreeParser.test.ts

```diff
@@ -97,10 +98,6 @@ describe('LayoutTreeParser', () => {
       expect(result.children[2].children[0].type).toEqual(LayoutType.Component);
     });
 
-    it('side menu center is require', () => {
```

Comment:
Why remove this test?

Reply:
Because according to the docs `center` is required https://wix.github.io/react-native-navigation/#/docs/layout-types?id=sidemenu

That is why I changed wrong type in TS
![image](https://user-images.githubusercontent.com/12229968/50476193-f2544500-09cf-11e9-9e01-8f77fa3adfe0.png)

Aka there is no need to test that if types are correct